src/main/java/com/google/sps/data/Room.java

```diff
@@ -40,6 +40,12 @@ public static Room createRoom(List<Member> members, Queue<Video> videos, Queue<M
         return new Room(members, videos, messages, null);
     }
```

Comment:
why this is needed. A parameter construction function is sufficient. A factory function as a wrapper do nothing more but increase the logic layer.